Makefile

```diff
@@ -51,7 +51,7 @@ preview-build: gen-content prereqs ## Builds a preview build (for e.g. a pull re
 branch-build: gen-content prereqs ## Builds a Git branch (for e.g. development branches).
 	hugo \
 		--baseURL $(DEPLOY_PRIME_URL) \
-		--buildDrafts \
+		--environment branch
```

Comment:
```suggestion
		--environment branch \
```

Reply:
We don’t need `buildDrafts` anymore?

Reply:
I assume we don't need them at this moment. When you deploy a branch you don't need to mark anything as draft because it's not deployed at fluxcd.io, anyway.